src/api/app.js

```diff
@@ -237,12 +237,12 @@ module.exports.authoriseShareMd = (appHandle, permissions) => {
 };
 
 /**
- * Lookup a given `safe://`-URL in accordance with the
+ * Lookup a given `safe://...` URL in accordance with the
  * convention and fetch the requested object.
  * @name window.safeApp.webFetch
  *
  * @param {SAFEAppHandle} appHandle the app handle
- * @param {AuthURI} authUri granted auth URI
+ * @param {String} url url to fetch
```

Comment:
Hi @loziniak, this change in the type of the param will actually break the link that is generated automatically in the documentation to the AuthURI type: http://docs.maidsafe.net/beaker-plugin-safe-app/#authuri
Can you please fix that and we can merge it.

Reply:
But AuthURI description reads `The auth URI ('safe-auth://...') returned by the Authenticator` - so it can be misleading for a parameter that has to begin with `'safe://'`.

Reply:
Oh, yes, you are absolutely right!, I think we need to add an additional definition for `safe://` URL's and reference it from here, perhaps a `safeURL` type?

Reply:
I think it's not necessary, since it's the only place where it is used. And url's format is described clearly enough in function's description IMO. Hence I would stick to String :-)